src/components/text-editor/__internal__/toolbar/toolbar.component.js

```diff
@@ -59,89 +58,132 @@ const Toolbar = ({
         setTabbable(true);
       } else if (Events.isLeftKey(ev)) {
         if (focusIndex === null || focusIndex === 0) {
+          controlRefs[3].current.focus();
```

Comment:
Is this all unrelated to tooltips as well?

Reply:
Yeah it was causing a bug in the tooltip behaviour so needed to be done either in this PR or be merged before it at least